### DISCUSSION TITLE
Exact area weighting in zonal statistics

### DISCUSSION THREAD

**Discussion by alaws-USGS** [corteva/geocube#125] [General]
Hello, I wondering how exact the rasterizing of a polygon using `make_geocube` was when performing zonal statistics? Specifically, I'm wondering how closely the `groupby` does area weighting to the original polygon boundaries when a pixel from a raster has area in multiple polygons. Thank you.

**Comment by snowman2** [comment]
It only considers the raster cell and does not consider the polygon. You can adjust the accuracy by using a smaller resolution for the grid.

**Reply by alaws-USGS** [reply]
Thank you for the information. Keep up the great work!
